.github/ISSUE_TEMPLATE/issue-report.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Issue report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+> Delete this text before submitting
+
+For best results from devs, please follow this guidance:
+
+1. Run your examples with the latest version (`remotes::install_github("JanMarvin/openxlsx2"`)
+1. Utilize the [`reprex` package](https://reprex.tidyverse.org/) and create a **minimal** reproducible example
```

Comment:
Should the counter go up or is that handled by markdown? Otherwise 👍

Reply:
Yup.  MD handles the tedious task of counting

Reply:
Nice :)